src/main.cpp

```diff
@@ -1,5 +1,6 @@
 #include <vector>
 #include <iostream>
+#include "functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
 #include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
```

Comment:
You have included functions in previous line, you can delete this line and comment above.